packages/compiler/test/experimental/typekit/define-kit.test.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, it } from "vitest";
+import { $, defineKit } from "../../../src/experimental/typekit/define-kit.js";
+
+it("can define a kit", async () => {
+  defineKit({
+    test() {
+      return true;
+    },
+  });
+
+  expect(($ as any).test()).toBe(true);
```

Comment:
to me this is one of the major issue with the typekit like that, dealing with patching a shared object and getting the right types is a mess

Reply:
Discussed offline with Tim, we'll look into alternatives. In the meantime we'll start with this